arduino-ide-extension/src/browser/widgets/cloud-sketchbook/cloud-sketchbook-widget.ts

```diff
@@ -54,4 +75,28 @@ export class CloudSketchbookWidget extends SketchbookWidget {
     });
     super.onAfterAttach(msg);
   }
+
+  registerCommands(registry: CommandRegistry): void {
+    this.sketchbookTreesContainer.addWidget(
+      this.cloudSketchbookCompositeWidget
+    );
+    registry.registerCommand(
+      CloudSketchbookCommands.SHOW_CLOUD_SKETCHBOOK_WIDGET,
+      {
+        execute: () => this.showCloudSketchbookWidget(),
+      }
+    );
+  }
+
+  showCloudSketchbookWidget(): void {
+    if (this.arduinoPreferences['arduino.cloud.enabled']) {
+      this.shell.activateWidget(this.id).then((widget) => {
+        if (widget instanceof CloudSketchbookWidget) {
+          widget.activateTreeWidget(this.cloudSketchbookCompositeWidget.id);
+        }
+        if (this.editorManager.currentEditor)
```

Comment:
Please use

```
if (foo) {
  bar();
}
```

instead of 
```ts
if (foo) bar()
```

Thank you!


----
Do not activate the editor. Thanks!

Reply:
I'm activating the current editor to ensure the sketchbook will show the current file node in the tree. Do you know another way to achieve this?

Reply:
> will show the current file node in the tree

You can select the tree node too.

Reply:
I did it calling `selectWidgetFileNode` from the `sketchbookWidgetContribution`.
 https://github.com/arduino/arduino-ide/pull/1102/files#diff-34e4625218827959b97fd2f2905e68bc3c513e02fd972a65a283176d9245136aR101-R103

is it fine to you?